src/seedu/addressbook/commands/SortingCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Sorts all persons in the address book lexicographically.
+ */
+
+public class SortingCommand extends Command{
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book.\n"
```

Comment:
Possible to add more details regarding the sorting? e.g. lexicographically by their names